docs/pipeline.rst

```diff
@@ -1,6 +1,40 @@
 QC and Analysis Pipeline
```

Comment:
Looks good! My comments are nitpicks:

line 4 seems like a run-on sentence, maybe drop "production" and end with "which produces publicly available data". Or really just remove the second phrase "that is used..."
line 8 missing o in electro
line 20 Oxford comma please =) 
line 11-23 maintain same tense and voice (e.g. use extracts, computes, and performs or extract, compute, and perform)
line 36 say "the feature extractor" or "IPFX", "the IPFX" sounds a bit weird to me
line 39 "will invoke"

Reply:
good catches. I have corrected the wording.